Examples/ObjectiveC/AnimatedLineExample.m

```diff
@@ -45,16 +45,13 @@ - (void)addLayer {
     
     // Add a layer to style our polyline.
     MGLLineStyleLayer *layer = [[MGLLineStyleLayer alloc] initWithIdentifier:@"polyline" source:source];
-    layer.lineJoin = [MGLStyleValue valueWithRawValue:[NSValue valueWithMGLLineJoin:MGLLineJoinRound]];
-    layer.lineCap = [MGLStyleValue valueWithRawValue:[NSValue valueWithMGLLineCap:MGLLineCapRound]];
-    layer.lineColor = [MGLStyleValue valueWithRawValue:[UIColor redColor]];
-    layer.lineWidth = [MGLStyleValue valueWithInterpolationMode:MGLInterpolationModeExponential
-        cameraStops: @{
-            @14: [MGLStyleValue valueWithRawValue:@5],
-            @18: [MGLStyleValue valueWithRawValue:@20]
-        }
-        options:@{MGLStyleFunctionOptionDefaultValue:@1.75}];
-
+    layer.lineJoin = [NSExpression expressionForConstantValue:[NSValue valueWithMGLLineJoin:MGLLineJoinRound]];
+    layer.lineCap = [NSExpression expressionForConstantValue:[NSValue valueWithMGLLineCap:MGLLineCapRound]];
```

Comment:
Do you think it’s better to stick with enumerations like `MGLLineCap` or use string literals like `round`? Both are supported inside expressions.

Reply:
From a newbie user's perspective, I think sticking to `MGLLineCapRound` would be clearer.